scripts/post_leap.sh

```diff
@@ -35,4 +35,11 @@ if [[ ! -f "${UPGRADE_LEAP_MARKER_FOLDER}/deploy-rpc.complete" ]]; then
 else
   log "deploy-rpc" "skipped"
 fi
+
+if [ -f "$UPGRADE_ELASTICSEARCH" ]; then
```

Comment:
This condition is looking for a file is that what we want? It seems better if we have this be a Boolean. Something like:

``` shell
export UPGRADE_ELASTICSEARCH=false


if [ "${UPGRADE_ELASTICSEARCH}" = true ]; then
    pushd /opt/rpc-upgrades/scripts
        source ./elasticsearch_upgrade.sh
    popd
fi
```

As a Boolean we'd not need to worry about string casting and cases.

Reply:
fixed.